### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by frankrousseau**
**Problem**

* Annotation group pasting is broken
* When a video with the wrong number of frames is uploaded, there is no warning.
* Following lucide-vue additions, frame number import icon and bot icon

**Solution**

* Ungroup pasted annotations
* Show a warning when the duration of uploaded videos doesn't match the number of frames information stored at the shot level
* Choose file digit and bot icons from lucide-vie to replace other icons

### CODE REVIEW

File: src/components/widgets/ButtonSimple.vue [hunk 1]
```diff
@@ -73,6 +79,7 @@ import {
   CornerLeftDownIcon,
   CornerRightDownIcon,
   DownloadIcon,
+  FileDigit,
```
Comment (NicoPennec) [inline src/components/widgets/ButtonSimple.vue:82]: Use `FileDigitIcon` to be consistent with other icon imports
```suggestion
  FileDigitIcon,
```

File: src/components/widgets/ButtonSimple.vue [hunk 2]
```diff
@@ -116,6 +123,7 @@ export default {
     DownloadIcon,
     EditIcon,
     Edit2Icon,
+    FileDigit,
```
Comment (NicoPennec) [inline src/components/widgets/ButtonSimple.vue:126]:
```suggestion
    FileDigitIcon,
```

File: src/components/widgets/ButtonSimple.vue [hunk 3]
```diff
@@ -45,6 +45,12 @@
     <codepen-icon class="icon" v-else-if="icon == 'codepen'" />
     <link-icon class="icon" v-else-if="icon == 'link'" />
     <clock-icon class="icon" v-else-if="icon == 'clock'" />
+    <file-digit
```
Comment (NicoPennec) [inline src/components/widgets/ButtonSimple.vue:48]: Use `file-digit-icon` to be consistent with other icon imports.
It is helpful to quickly find all the `*-icons` components.

```suggestion
    <file-digit-icon
```

File: src/locales/en.js [hunk 4]
```diff
@@ -1387,6 +1387,7 @@ export default {
     sequences: 'Sequences',
     tasks: 'Shot Tasks',
     title: 'Shots',
+    wrong_file_duration: 'One of the upldoaded video file duration doesn\'t match the expected duration of the current shot.',
```
Comment (NicoPennec) [inline src/locales/en.js:1390]: Add variants in `en_nft.js` and `en_video-games.js`

> shot = NFT
> shot = map

**Comment by EvanBldy** [comment]
Thanks

**Pull request merged**